README.md

```diff
@@ -1,4 +1,4 @@
-﻿<!-- markdownlint-disable-next-line -->
+<!-- MARKDOWNLINT-DISABLE-NEXT-LINE -->
```

Comment:
```suggestion
<!-- markdownlint-disable-next-line -->
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix